app/src/main/java/com/everest/movieapp/SearchScreenActivity.kt

```diff
@@ -0,0 +1,63 @@
+package com.everest.movieapp
+
+import android.os.Bundle
+import android.view.Menu
+import android.widget.SearchView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.RecyclerView
+import com.everest.movieapp.adapters.MovieRecyclerViewAdapter
+import com.everest.movieapp.api.Api
+import com.everest.movieapp.api.RetrofitHelper
+import com.everest.movieapp.databinding.ActivitySearchScreenBinding
+import com.everest.movieapp.model.MovieDb
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class SearchScreenActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivitySearchScreenBinding
+    private lateinit var movieRecyclerViewAdapter: MovieRecyclerViewAdapter
+    private lateinit var recyclerView: RecyclerView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySearchScreenBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        val api: Api = RetrofitHelper.getInstance().create(Api::class.java)
+        val currentYearMovieData = api.getPopularViews()
+        recyclerView = binding.searchTv
+
+        currentYearMovieData.enqueue(object : Callback<MovieDb> {
```

Comment:
Here we can extract ViewModel and Repository

This app might crash on configuration changes( loading data, but instances may be null somtimes)